index.js

```diff
@@ -142,16 +142,24 @@ const _onBeforeInput = shortcutsOfWindow => (e, input) => {
 };
 
 /**
-* Registers the shortcut `accelerator`on the BrowserWindow instance.
+ * Registers the shortcut `accelerator`on the BrowserWindow instance.
  * @param  {BrowserWindow} win - BrowserWindow instance to register.
  * This argument could be omitted, in this case the function register
  * the shortcut on all app windows.
- * @param  {String} accelerator - the shortcut to register
+ * @param  {String|Array<String>} accelerator - the shortcut to register
  * @param  {Function} callback    This function is called when the shortcut is pressed
  * and the window is focused and not minimized.
- * @return {Undefined}
+ * @return {void}
```

Comment:
Why you used void here? Undefined is a better choice.

Reply:
Sorry, my fault, I'm using void when I'm trying to tell that "don't expect anything from this function". I must follow already defined convention.